salesagility/SuiteCRM
# API - Add Get module list and get module field list endpoints

Adds two new endpoints to the API.

## Description




`V8/meta/modules`
Returns a list of modules

`V8/meta/fields/{moduleName}`
Returns a list of fields for this module, with field types

## Motivation and Context

Fix #7174 
Fix #7175 

## How To Test This

1. GET `V8/meta/modules` and see it returns a complete list of modules
2. GET `V8/meta/fields/Accounts` and see it returns all the fields for this module 

## Types of changes


### Final checklist
